packages/cw-orch-mock/src/queriers/env.rs

```diff
@@ -9,10 +9,11 @@ impl<A: Api, S: StateInterface> EnvironmentQuerier for MockBase<A, S> {
     fn env_info(&self) -> EnvironmentInfo {
         let block_info = self.block_info().unwrap();
         let chain_id = block_info.chain_id.clone();
-        let chain_name = chain_id.rsplitn(2, '-').collect::<Vec<_>>()[1].to_string();
+        let chain_name = chain_id.rsplitn(2, '-').last().unwrap().to_string();
+
         EnvironmentInfo {
             chain_id,
-            chain_name,
+            chain_name: chain_name.clone(),
```

Comment:
```suggestion
            chain_name,
```
No need to clone it anymore